src/libs/ajax/file-browser-providers/GCSFileBrowserProvider.ts

```diff
@@ -34,7 +34,7 @@ const GCSFileBrowserProvider = ({ bucket, project, pageSize = 1000 }: GCSFileBro
 
     if (nextPageToken || isFirstPage) {
       do {
-        const requestOptions: { maxResults: number, pageToken?: string } = {
+        const requestOptions: { maxResults: number; pageToken?: string } = {
```

Comment:
is this intentional?
I think we want to stick with the more standard comma delimiter in json-style object notation cases like this.